src/blocks/packages/apt.rs

```diff
@@ -0,0 +1,179 @@
+//! Pending updates available for your Debian/Ubuntu based system
```

Comment:
We don't usually put block specific usage notes in the submodule (apt/pacman), just in the packages' docs

Reply:
Done